server/automute.go

```diff
@@ -159,8 +160,20 @@ func (p *Plugin) canAutomuteChannelID(channelID string) (bool, error) {
 // canAutomuteChannel returns true if the channel is either explicitly linked to a channel in MS Teams or if it's a
 // DM/GM channel that is implicitly linked to MS Teams.
 func (p *Plugin) canAutomuteChannel(channel *model.Channel) (bool, error) {
-	// Automute all DM/GM channels
-	if channel.IsGroupOrDirect() {
+	// Automute all GM channels
+	if channel.Type == model.ChannelTypeGroup {
+		return true, nil
+	} else if channel.Type == model.ChannelTypeDirect {
+		userIDs := strings.Split(channel.Name, "__")
```

Comment:
I was surprised we didn't expose this from the `model` package! Maybe a future addition :)